### DISCUSSION TITLE
Add new column for sum of bytes in Visualize results

### DISCUSSION THREAD

**Discussion by pk-lee-developer** [akvorado/akvorado#933] [Q&A]
Can you add a new column for total sum of bytes, instead just an average of all bytes in future release?

We are try to monitor how much total bytes used by each source AS in a monthly report.

Thanks.

**Comment by pk-lee-developer** [comment]
Is it possible to add download to CSV and/or pdf (with graph) option in Visualize?

**Reply by vincentbernat** [reply]
Currently, no. But you can still copy/paste the table in LibreOffice and Excel and then export to CSV.

**Comment by pk-lee-developer** [comment]
It appears that "visualize" already had a column for "average".  So it  already has the calculation for sum then divide for average. Can we have the sum value in a separate column?

**Comment by joseph-holland** [comment]
I've recently installed Akvorado to try and help diagnose what clients on my network are using all my data cap.  I've played around with the reports, but it seems that it doesn't support summing total bytes, correct; only average and grouping into packet size buckets.
Putting this comment here, as it seemed like it was very related.
